src/devices/bus/coco/coco_t4426.cpp

```diff
@@ -0,0 +1,195 @@
+// license:BSD-3-Clause
+// copyright-holders:Joakim Larsson Edstrom
+/***************************************************************************
+ *
+ *  coco_t4426.c
+ *
+ *  Terco T4426 CNC Programming Station multi cart
+ *
+ *  The code here is heavily inspired by coco_pak and coco_232
+ *
+ *  +-------------------------------------------------------------------------------+
+ *  ||__|+-----+    oo   75                               |O ||||||||||||||| O|     |
+ *  |XTL||MC   |    oo  110                                                         |
+ *  |1.8||14411|    oo  ..        +--+ +--+                                         |
+ *  |432||BRG  |    ==  600       |74| |74| +------+                                |
+ *  |MHz||     |    oo  ..        |LS| |LS| |MC1488|                                |
+ *  +---+|     |    oo 7200       |139 |00| +------+                                |
+ *  |    +-----+    oo 9600       +--+ +--+                    +--+                 |
+ *  |                                      +-------------+     |MC|                 |
+ *  |    +-----+   +-----+    +-----+      | EF68B50P    |     |14|                 |
+ *  |    | 2764|   | 2764|    |     |      | ACIA        |     |89|                 |
+ *  |    |     |   |     |    |     |      +-------------+     +--+                 |
+ *  |    |CA   |   |CA   |    |PMOS |                                               |
+ *  |    | 4426|   | 4426|    | 4426|   +-------------------+                       |
+ *  |    |  -6 |   |  -7 |    |     |   |  EP68B21P         |                       |
+ *  |    |     |   |     |    |     |   |  PIA              |                       |
+ *  |    |     |   |     |    +-----+   +-------------------+                       |
+ *  |    +-----+   +-----+                                                          |
+ *  |    +-----+   +-----+    +-----+   +-----+    +-----+   +-----+                |
+ *  |    | 2764|   | 2764|    | 2764|   | 2764|    | 2764|   | 2764|                |
+ *  |    |     |   |     |    |     |   |     |    |     |   |     |                |
+ *  |    |CA   |   |CA   |    |PD   |   |PD   |    |ED   |   |ED   |                |
+ *  |    | 4426|   | 4426|    | 4426|   | 4426|    | 4426|   | 4426|                |
+ *  |    |  -5 |   |  -4 |    |  -3 |   |  -2 |    |  -1 |   |  -0 |                |
+ *  |    |     |   |     |    |     |   |     |    |     |   |     |       OO       |
+ *  |    |     |   |     |    |     |   |     |    |     |   |     |                |
+ *  |    +-----+   +-----+    +-----+   +-----+    +-----+   +-----+                |
+ *  +-------------------------------------------------------------------------------+
+ *
+ ***************************************************************************/
+
+#define VERBOSE 0
```

Comment:
Can you please move the local macros after you #include headers?  It's bad form to inject stuff into the global namespace before including headers, and it will make for strange behaviour with/without the precompiled emu.h prefix header if you've got any code in your translation unit before you #include emu.h